win32_build.sh

```diff
@@ -162,7 +162,7 @@ cmake -D CMAKE_PREFIX_PATH=$depsDirs \
       -D WITH_GSSAPI=OFF -D WITH_XFS=OFF \
       -D WITH_FUSE=OFF -D WITH_DOKAN=ON \
       -D WITH_BLUESTORE=OFF -D WITH_LEVELDB=OFF \
-      -D WITH_LTTNG=OFF -D WITH_BABELTRACE=OFF \
+      -D WITH_LTTNG=OFF -D WITH_BABELTRACE=OFF -D WITH_JAEGER=OFF \
```

Comment:
You might want to add this also to the ./`do_freebsd.sh` somewhere arouund line 175 where it fits with the other options

Reply:
sure, thanks!